buildcmake

```diff
@@ -196,9 +228,13 @@ while [[ $# -gt 0 ]]; do
       ;;
 
   # Miscellaneous options
-    -j*)
+    -j[0-9]*)
       opt_parallel=$arg
       ;;
+    -j)
+      opt_parallel="$arg$1"
```

Comment:
`-j` a valid parameter by itself (without a number following it). `$arg$1` will unconditionally eat the argument following `-j`.

Reply:
Good point. Should be fixed now.